tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -90,6 +94,7 @@ internal static void CleanLogFiles(int deleteAfter, string logsDirectory)
               {
                 "dotnet-tracer-*.log",
                 "dotnet-native-loader-*.log",
+                "dotnet-libdatadog-*.log",
```

Comment:
Are the logs going to be strictly this, or should they be `dotnet-tracer-libdatadog-*` (seeing as we have `dotnet-tracer-managed` and `dotnet-tracer-native` for example